projects/examples/src/components/datagrid/datagrid.examples.module.ts

```diff
@@ -54,6 +56,11 @@ Documentation.registerDocumentationEntry({
             forComponent: null,
             title: 'Select datagrid row example',
         },
+        {
+            component: DatagridPaginationExampleComponent,
+            forComponent: null,
+            title: 'Shows the pagination capability of the datagrid',
```

Comment:
Remove the word shows. We always want to make messages concise and avoid repetition. Can we be more specific about which of the features? Custom text for blah and blah?